### ISSUE TITLE
Right atrial thrombus

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6169]
**Preferred term label:**
Right atrial thrombus

**Synonyms**
Right atrial thrombosis

**Definition (free text, please give PubMed ID)**
The formation of a blood clot inside a vein of the right atrial; PMID: 28127133

**Parent term (use hpo.jax.org/app)**
Venous thrombosis

**Comment by pnrobinson** [comment]
added as HP:0033138
